extension/content.css

```diff
@@ -288,6 +288,30 @@ svg.octicon.octicon-star.dashboard-event-icon {
 	color: inherit !important;
 }
 
+/* Created repository style in dashboard */
```

Comment:
@AshikNesin You're using spaces here, and this project uses tabs. Would you mind changing it over? I'm not sure why it's not throwing an error with `xo`. (@sindresorhus?)

Otherwise this PR is good!